modules/aws_documentdb/aws-documentdb.yaml

```diff
@@ -34,6 +34,11 @@ inputs:
     validator: bool(required=False)
     description: A value that indicates whether the DB cluster has deletion protection enabled. The database can't be deleted when deletion protection is enabled. By default, deletion protection is disabled.
     default: false
+  - name: identifier
```

Comment:
For all these identifier strings, do we need input validation?

For example, forbidden names of clusters, too long/too short, special characters not allowed etc. There may be some constraints on these...not

Reply:
I'm not sure whether we should put validations or not on these.
Currently I left the validation to individual terraform or provider restrictions.